deb/debian/stackdriver-agent.init

```diff
@@ -35,7 +35,7 @@ _PIDFILE=/var/run/collectdmon.pid
 JAVA_LIB_DIR=""
 
 # Gracefully exit if the package has been removed.
-test -x "$DAEMON" || exit 0
+test -x "$COLLECTDMON" || exit 0
```

Comment:
We want both here, actually — if the daemon executable is not present, we want to exit early as well.

Reply:
Good point, modifying!